pom.xml

```diff
@@ -728,6 +728,7 @@ limitations under the License.]]></inlineHeader>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
         <configuration>
+          <jvm>${testing.jvm}/bin/java</jvm>
```

Comment:
Earlier work on the PR for JAVA-3042 had added the "jvm" param to all subprojects that had their own pom.xml... but that didn't cover anything that inherited settings from the parent POM.